src/pyodide.js

```diff
@@ -148,8 +148,11 @@ var languagePluginLoader = new Promise((resolve, reject) => {
         messageCallback(`Loading ${packageList}`);
       }
 
+      var packageCounter = Object.keys(toLoad).length * 2;
+
       window.pyodide._module.monitorRunDependencies = (n) => {
```

Comment:
you don't need the `n` argument anymore here

Reply:
Good point!